data/json/recipes/electronics/parts.json

```
@@ -119,6 +119,22 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "scrap", 6 ] ], [ [ "cable", 4 ] ], [ [ "plut_cell", 20 ] ] ]
   },
+  {
+    "result": "flux_comp_gen",
+    "type": "recipe",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_PARTS",
+    "skill_used": "electronics",
+    "difficulty": 5,
+    "skills_required": [ "fabrication", 4 ],
+    "time": "60 m",
+    "decomp_learn": 4,
+    "book_learn": [ [ "textbook_anarch", 5 ], [ "advanced_electronics", 5 ] ],
+    "using": [ [ "soldering_standard", 10 ] ],
+    "qualities": [ { "id": "SCREW", "level": 1 } ],
+    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "forge", 25 ] ] ],
+    "components": [ [ [ "cu_pipe", 1 ] ], [ [ "cable", 77 ] ], [ [ "composite_b", 450 ] ], [ [ "concrete", 1 ] ] ]
```

To match the values I found for weight for these.
```suggestion
    "components": [ [ [ "cu_pipe", 2 ] ], [ [ "cable", 248 ] ], [ [ "composite_b", 450 ] ], [ [ "concrete", 1 ] ] ]
```